config/locales/find.yml

```diff
@@ -302,6 +302,13 @@ en:
         label:
           one: '%{count} mile'
           other: '%{count} miles'
+  helpers:
```

Comment:
Why are we putting these "translations" under `'helpers.legend'`?

Reply:
To follow the same pattern as https://govuk-form-builder.netlify.app/building-blocks/localisation/

The reason I couldn't use the above is because govuk-form-builder don't translate if the `scope: ''` on the `form_with`.